classes/UpgradeContainer.php

```diff
@@ -415,19 +416,28 @@ public function getTranslator()
     }
 
     /**
-     * @return Twig_Environment
+     * @return Twig_Environment|\Twig\Environment
      */
     public function getTwig()
     {
         if (null !== $this->twig) {
             return $this->twig;
         }
 
-        // Using independant template engine for 1.6 & 1.7 compatibility
-        $loader = new Twig_Loader_Filesystem();
-        $loader->addPath(realpath(__DIR__ . '/..') . '/views/templates', 'ModuleAutoUpgrade');
-        $twig = new Twig_Environment($loader);
-        $twig->addExtension(new TransFilterExtension($this->getTranslator()));
+        if (class_exists(\Twig\Environment::class)) {
```

Comment:
I'm not sure about that because the class `\Twig\Environment` also exists in twig 1.x

Reply:
You're right :O 
https://github.com/twigphp/Twig/blob/1.x/src/Environment.php#L1638

Reply:
@atomiix Thanks. I have fixed with a reverse test.